<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-4VBBEJ4" failures="1" tests="1" name="org.runner.TestRunnerClass" time="3.054" errors="0" timestamp="14 Aug 2023 06:14:57 GMT" skipped="0">
  <testcase classname="org.runner.TestRunnerClass" name="runScenario" time="3.054">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at org.steps.StepDefintionClass.user_clicks_on_the_login_button(StepDefintionClass.java:30)
at âœ½.User clicks on the login button(file:///C:/Users/LENOVO/newEclipseWorkspe/CucumberTestng/src/test/resources/Facebook.feature:5)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- org.runner.TestRunnerClass -->
